Admin/FormAdmin.php

```diff
@@ -75,9 +156,15 @@ public function getSecurityContexts()
         return [
             'Sulu' => [
                 'Form' => [
-                    'sulu.form.forms',
+                    'sulu.form.forms' => [
```

Comment:
/cc @chirimoya this seems to be a BC Break that it need that permissions need to be defined